fe/src/app.tsx

```diff
@@ -28,8 +28,8 @@ export function App() {
           <PageWrapper path="/:lang/article">
             {(props: PageProps) => <ArticleListPage {...props} />}
           </PageWrapper>
-          <PageWrapper path="/:lang/article/:locationId/:articleId">
-            {(props: PageProps) => <PeopleDetailPage {...props} />}
+          <PageWrapper path="/:lang/article/:articleId">
```

Comment:
Just noticed this change. I put a `locationId` there in purpose to make sure cases like a shared link (not containing session states or routing history) can know its parent and route to the correct path when hitting back button. When `?fromPeople=1`, it just ignore that and go directly to people route.

Make sure you find a way to handle this case before removing the router parameter.

Reply:
ok